### PR TITLE
ocaml-findlib: Update to 1.9.1

### PR DESCRIPTION
**Pull request by ryandesign**
#### Description

ocaml-findlib: Update to 1.9.1

I don't know anything about ocaml so I don't know if my changes are correct but I hope they are. Perhaps someone who knows something about ocaml can verify them.

I deleted patch-meta.diff because it failed to apply. Its purpose was apparently to "Avoid installing META files for camlp4, labltk, and num." Even without this patch, 1.9.1 installs only these META files:

```
$ port contents ocaml-findlib | grep /META$
  /opt/local/lib/ocaml/site-lib/bigarray/META
  /opt/local/lib/ocaml/site-lib/bytes/META
  /opt/local/lib/ocaml/site-lib/compiler-libs/META
  /opt/local/lib/ocaml/site-lib/dynlink/META
  /opt/local/lib/ocaml/site-lib/findlib/META
  /opt/local/lib/ocaml/site-lib/ocamldoc/META
  /opt/local/lib/ocaml/site-lib/stdlib/META
  /opt/local/lib/ocaml/site-lib/str/META
  /opt/local/lib/ocaml/site-lib/threads/META
  /opt/local/lib/ocaml/site-lib/unix/META
```

When configuring, it prints this output:

```
num: not present (normal since 4.06)
```
```
labltk: not present (normal since 4.02)
```
```
camlp4: not present (normal since OCaml-4.02)
```

I take this to mean that the patch is no longer needed.

I removed camlp4find from the files directory because it was unused. I removed the commented-out code in the portfile that had previously installed this file.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.6 17G14042 x86_64
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: ocaml/ocaml-findlib/Portfile [hunk 1]
```diff
@@ -30,25 +34,18 @@ long_description    The findlib library provides a scheme to manage reusable \
 homepage            http://projects.camlcity.org/projects/findlib.html
 master_sites        http://download.camlcity.org/download/
```
Comment (pmetzger) [inline ocaml/ocaml-findlib/Portfile:35]: We should probably use Github as the master site at this point for downloads, given that camlcity is essentially dead. Yes, I know the project page on github points to camlcity.

**Review by pmetzger — CHANGES_REQUESTED** [review]
Looks mostly good to me. I presume you're correct about the META files no longer needing to be patched out. I'd make this use Github for downloads and then you should feel free to commit it at will.

**Comment by reneeotten** [comment]
@pmetzger do you want to push these changes to @ryandesign his branch and merge this PR?

**Comment by pmetzger** [comment]
I suppose I could, though at that point perhaps I should just update the port myself? Anyway, you're right that this does require attention.

**Comment by pmetzger** [comment]
The patch to leave out the camlp4 META file is still needed, FWIW. Regardless, I'm now finishing up a patched version of all of this.

**Comment by pmetzger** [comment]
Okay, I've committed a new version of the package by hand in 933282c07b2 ; thank you @ryandesign for your fixes.

Note that the new package on github seems to be missing the documentation; I have an issue opened for that.

**Pull request closed without merging**